server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -43,6 +43,9 @@ export class SyncChefsDataService {
       submission: responseData.submission.data,
       confirmationId: responseData.confirmationId,
       facilityName: responseData.submission.data.facilityName,
+      projectTitle: responseData.submission.data.projectTitle,
+      totalEstimatedCost: responseData.submission.data.totalEstimatedCostOfProject,
+      asks: responseData.submission.data.totalRequestBeingMadeOfBcaapACDNotToExceedB,
       assignedTo: null,
       status: ReviewStatuses.INITIAL_REVIEW,
```

Comment:
Similarly here. Set the default value to be initial review in the entity and remove this status from here.